src/v2/Apps/Order/Components/__tests__/PaymentPicker.jest.tsx

```diff
@@ -230,32 +230,6 @@ describe("PaymentPickerFragmentContainer", () => {
     expect(page.addressFormIsVisible).toBe(true)
   })
 
-  it("removes all data when the billing address form is hidden", async () => {
```

Comment:
@lilyfromseattle Removing the whole test because we don't remove the data anymore when the billing address form is hidden.